terraform/swodlr-main.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_s3_bucket" "swodlr-site-bucket" {
+  bucket = local.ec2_resources_name
```

Comment:
Given the other suggested changes, the name of the s3 bucket here would end up being `podaac-services-sit-swodlr-ui`. Which is fine, just making the comment to hopefully help clear things up a bit more.